pkg/oci/oci.go

```diff
@@ -80,9 +80,9 @@ const (
 	localhost = "localhost"
 )
 
-// getLocalOciStore returns a single local oci store. oci.Store is concurrently safe only
+// GetLocalOciStore returns a single local oci store. oci.Store is concurrently safe only
 // against its own instance inside the same go program
-var getLocalOciStore = sync.OnceValues(func() (*oci.Store, error) {
+var GetLocalOciStore = sync.OnceValues(func() (*oci.Store, error) {
```

Comment:
Maybe a function would be better than an exported var that could be changed on runtime.